bin/README.md

```diff
@@ -0,0 +1,3 @@
+# directory `bin`
+
+This directory contains tools for operating pulldasher
```

Comment:
Heh...  I really don't think `/bin` needs an explanation. It's effectively a linux / unix standard.

Reply:
I guess so 🤷🏼‍♂️     I don't think it hurts though for those who may not be familiar with the convention.

Reply:
Also, the main reason for this is when you're browsing GitHub, it displays the README.md in each directory when viewing directories.